prow/apis/prowjobs/v1/types.go

```diff
@@ -196,6 +196,14 @@ type ProwJobSpec struct {
 	// ProwJobDefault holds configuration options provided as defaults
 	// in the Prow config
 	ProwJobDefault *ProwJobDefault `json:"prowjob_defaults,omitempty"`
+
+	// JobQueueName is an optional field with name of a queue defining
+	// max concurrency. When several jobs from the same queue try to run
+	// at the same time, the number of them that is actually started is
+	// limited by JobQueueConcurrencies (part of Plank's config). If
+	// this field is left undefined or the queue name is not present in
+	// Plank's config, inifinite concurrency is assumed.
+	JobQueueName string `json:"job_queue_name,omitempty"`
```

Comment:
There is currently no way to configure this field for a job because it does not exist in the job configuration, only the ProwJob resource. We need to add this field to configuration (specifically `JobBase`) and make sure it is copied to the ProwJob resource when [creating the ProwJobSpec](https://github.com/kubernetes/test-infra/blob/61275e2d9c3f4acad2cd5383f1ad174e9078108d/prow/pjutil/pjutil.go#L192). You can grep for `MaxConcurrency` to see how this should be done.

Reply:
Fixed in https://github.com/kubernetes/test-infra/pull/25674/commits/6cf071713ef2ccd31473141cb753e2528d26e202.